enterprise/backend/test/metabase_enterprise/semantic_search/core_test.clj

```diff
@@ -81,8 +82,21 @@
                     (is (not @semantic-called?) "Semantic search should not be called when overridden")
                     (is (not @appdb-called?) "AppDB search should not be called when overridden")))))))))))
 
-(def ^:private search-context
-  {:search-string "test" :search-engine :search.engine/semantic})
```

Comment:
At best having a search context that violates the type constraints was causing a lot of noise pollution in tests.

In the worst case it could lead to undefined or misleading behavior.